packages/system/src/Managed/releaseMap.ts

```diff
@@ -8,6 +8,8 @@ export type Finalizer = (
   exit: T.Exit<any, any>
 ) => T.Effect<unknown, unknown, never, any>
 
+export const noopFinalizer: Finalizer = () => T.unit
```

Comment:
dupe of https://github.com/Matechs-Garage/matechs-effect/blob/master/packages/system/src/Managed/managed.ts#L4